bc_gym_planning_env/utilities/path_tools.py

```diff
@@ -405,7 +405,7 @@ def parallel_distances(pose, path_poses):
     return np.cos(path_poses[:, 2]) * (pose[0] - path_poses[:, 0]) + np.sin(path_poses[:, 2]) * (pose[1] - path_poses[:, 1])
 
 
-def find_reached_indices(pose, segment, spatial_precision, angular_precision, parallel_distance_threshold=None):
+def find_reached_indices(pose, segment, spatial_precision, angular_precision, parallel_distance_threshold=float('inf')):
```

Comment:
do you really have to fix this file?
you're changing default behavior for find_reached_indices. I realized that we don't have much tests in this repo yet, but the mainline tests might break:
before find_reached_indices default was -spatial_precision / 9 now its inf. So it does change the behavior in our benchmarks.

Can you revert this file and I'll try this fix separately in the mainline?

Reply:
Not really, I guess...... Just thought the logic was a little bit weird :) Since I don't have much context. I've reverted this back. @olegsinyavskiy